completion/available/hub.completion.bash

```diff
@@ -105,9 +107,9 @@ if _is_function _git && ! _is_function __git_list_all_commands_without_hub; then
 			esac
 			((c++))
 		done
-		if [ -z "$repo" ]; then
+		if [[ -z "$repo" ]]; then
```

Comment:
Does shellcheck complain about this?

Reply:
no, I just see no point in forking the external `/bin/[` when I have a built in shell command for it.

Reply:
`[` doesn't call the external command. `[` is also a Bash builtin. See the result of `type [`. That said, `[[` is generally recommended for Bash scripts (`[` would be used in portable scripts specified by POSIX).

Also, if shellcheck doesn't complain about this, isn't it considered refactoring? You told me that

> In an attempt to make the minimum needed number of changes to make shellcheck happy, I don't fix what it's not complaining about, otherwise this becomes a refactoring mission, and that's something for a future stage IMHO.

though I don't mind if you do some refactoring in the PR.

If you are going to do this in this PR, the same change should also be applied to all the other cases of `[ -z "..." ]` and `[ -n "..." ]` and also other calls of `[`.

Reply:
true. I'm only human.